Content/Data/Body-External.xml

```diff
@@ -130,11 +130,314 @@
+    <source>(0,0,60,60)</source>
+  </body>
+  <!--Armored Orc w Longsword and Shield-->
+  <body id="363" corpse="5362">
+    <texture>Tenser\monsters\armored_orc</texture>
+    <source>(60,0,60,60)</source>
+  </body>
+  <!--Armored Orc w Greatsword-->
+  <body id="364" corpse="5362">
+    <texture>Tenser\monsters\armored_orc</texture>
+    <source>(120,0,60,60)</source>
+  </body>
+  <!--Armored Orc w Crossbow-->
+  <body id="365" corpse="5362">
+    <texture>Tenser\monsters\armored_orc</texture>
+    <source>(0,60,60,60)</source>
+  </body>
+  <!--Armored Orc w Kopesh and Shield-->
+  <body id="366" corpse="5362">
+    <texture>Tenser\monsters\armored_orc</texture>
+    <source>(60,60,60,60)</source>
+  </body>
+  <!--Armored Orc with Bow-->
+  <body id="367" corpse="5362">
+    <texture>Tenser\monsters\armored_orc</texture>
+    <source>(120,60,60,60)</source>
+  </body>
+  <!--Orc Caster Basic (red)-->
+  <body id="368" corpse="5368">
+    <texture>Tenser\monsters\orc_casters</texture>
+    <source>(0,0,60,60)</source>
+  </body>
+  <!--Orc Caster Shaman-->
+  <body id="369" corpse="5345">
+    <texture>Tenser\monsters\orc_casters</texture>
+    <source>(60,0,60,60)</source>
+  </body>
+  <!--Orc Caster with Helm (red)-->
+  <body id="370" corpse="5368">
+    <texture>Tenser\monsters\orc_casters</texture>
+    <source>(120,0,60,60)</source>
+  </body>
+  <!--Orc Caster (black)-->
+  <body id="371" corpse="5371">
+    <texture>Tenser\monsters\orc_casters</texture>
+    <source>(0,60,60,60)</source>
+  </body>
+  <!--Basic Armored Orc Caster (Black)-->
+  <body id="372" corpse="5371">
+    <texture>Tenser\monsters\orc_casters</texture>
+    <source>(60,60,60,60)</source>
+  </body>
+  <!--Armored Orc caster (Black)-->
+  <body id="373" corpse="5371">
+    <texture>Tenser\monsters\orc_casters</texture>
+    <source>(120,60,60,60)</source>
+  </body>
+  <!--Orc Praetor Scimitar and Shield-->
+  <body id="374" corpse="5374">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(0,0,60,60)</source>
+  </body>
+  <!--Orc Praetor with Greataxe-->
+  <body id="375" corpse="5374">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(60,0,60,60)</source>
+  </body>
+  <!--Orc Praetor with Axe-->
+  <body id="376" corpse="5374">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(120,0,60,60)</source>
+  </body>
+  <!--Orc Praetorian Raider Longsword and Shield-->
+  <body id="377" corpse="5377">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(0,60,60,60)</source>
+  </body>
+  <!--Orc Praetorian Raider with Greatsword-->
+  <body id="378" corpse="5377">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(60,60,60,60)</source>
+  </body>
+  <!--Orc Praetorian Raider w Kopesh and Shield-->
+  <body id="379" corpse="5377">
+    <texture>Tenser\monsters\orc_praetorean</texture>
+    <source>(120,60,60,60)</source>
+  </body>
+=======
```

Comment:
Merge line endings are here. Need to clean those up and make sure the xml is valid.